metricq/history_client.py

```diff
@@ -445,8 +458,7 @@ async def history_aggregate(
             A single aggregate over values of this metric, including minimum/maximum/average/etc. values.
 
         Raises:
-            InvalidHistoryResponse:
-                if an invalid response was received
+            ~exceptions.InvalidHistoryResponse:
```

Comment:
It seems redundant, but every `raises` had a short description for when the exception is raised. I'd add it for consistency